puppet/modules/web/manifests/base.pp

```diff
@@ -10,6 +10,9 @@
   }
 
   include apache
+  if $facts['os']['family'] == 'RedHat' and $facts['os']['release']['major'] == '9' {
+    include apache::mod::expires
+  }
```

Comment:
Can be dropped once we have https://github.com/puppetlabs/puppetlabs-apache/pull/2559

Reply:
On line 61 in puppet/modules/freight/manifests/user.pp it mentions the headers module, but I think that should have been `apache::mod::expires`.

Also, why is this limited to EL9? I'd think this is needed everywhere.

Reply:
Needed? Yes. But on EL7 we have default mods enabled, and then you get duplicate definitions blah blah

Reply:
https://github.com/theforeman/foreman-infra/actions/runs/10596140550/job/29363435733

Reply:
Ah, probably because e2a182031f9f448f01f7fa262915936815d45108 changed `Header` to `use mod_expire`!